e2e-polybft/cardanofw/test-apex-bridge.go

```diff
@@ -306,6 +306,10 @@ func RunApexBridge(
 
 	fmt.Printf("Apex bridge setup done\n")
 
+	SetupAndRunNexusBridge(t, bladeValidatorsNum)
```

Comment:
Nexus is not a bridge. It is just a chain, similar like prime and vector. The only difference is that it is evm and not cardano. Rename all the structs and funcs accordingly. Nexus should be started before apex bridge (can be after  prime and vector)